shared/actions/fs/index.js

```diff
@@ -151,9 +152,14 @@ function* folderList(action: FsGen.FolderListLoadPayload): Saga.SagaGenerator<an
   // Get metadata fields of the directory that we just loaded from state to
   // avoid overriding them.
   const state = yield Saga.select()
-  const {lastModifiedTimestamp, lastWriter, size, writable, favoriteChildren, tlfMeta}: Types.FolderPathItem = state.fs.pathItems.get(
-    rootPath
-  )
+  const {
+    lastModifiedTimestamp,
+    lastWriter,
+    size,
+    writable,
+    favoriteChildren,
+    tlfMeta,
+  }: Types.FolderPathItem = state.fs.pathItems.get(rootPath)
```

Comment:
I think my editor linter made these changes, but it passed the precommit hooks so I didn't bother reverting it.